packages/common/components/breadcrumbs/marko.json

```diff
@@ -29,6 +29,9 @@
     "endeavor-breadcrumbs-contact-us": {
       "template": "./contact-us.marko"
     },
+    "endeavor-breadcrumbs-editorial-advisory-board": {
```

Comment:
This doesn't need to be a custom component. Use the standard website section breadcrumbs instead.

Reply:
If I change it to <endeavor-breadcrumbs-website-section />, it only displays "Home" in the breadcrumbs. If I change it to <endeavor-breadcrumbs />, it shows nothing.

Reply:
This doesn't work? `<endeavor-breadcrumbs-website-section section=section />`

Reply:
Ah, I didn't have the section=section. Pushing up now.

Reply:
See https://www.offshore-mag.com/resources/maps-posters as an example:

`/sites/offshore-mag/server/templates/website-section/resources/maps-posters`

Reply:
Coolio :)

Reply:
Perhaps need a follow-up ticket to the contact-us refactoring, to utilize the `<endeavor-breadcrumbs-website-section section=section />` instead of '<endeavor-breadcrumbs-contact-us />' custom component.

Reply:
Yes indeed. I missed that during the first refactor. :)

Reply:
Pushed contact-us update here: https://github.com/endeavorb2b/websites/pull/287